staging/src/k8s.io/sample-apiserver/pkg/admission/initializer/initializer.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package initializer
```

Comment:
The generic `initializer` names in this PR are confusing IMO. Can we rename it to `CustomInitializer`, `WardleInitializer`, `SampleInitializer` to make clear that we are defining something custom?